editor/editor_help_search.cpp

```diff
@@ -766,9 +793,26 @@ String EditorHelpSearch::Runner::_build_method_tooltip(const DocData::ClassDoc *
 		}
 	}
 	tooltip += ")";
+	tooltip += _build_keywords_tooltip(p_doc->keywords);
 	return tooltip;
 }
 
+String EditorHelpSearch::Runner::_build_keywords_tooltip(const String &p_keywords) const {
+	String tooltip;
+	if (p_keywords.is_empty()) {
+		return tooltip;
+	}
+
+	tooltip = "\n    " + TTR("Search Keywords") + ": ";
```

Comment:
I would suggest wholy different code for this but this may look better:
```suggestion
	tooltip = "\n\n" + TTR("Keywords") + ": ";
```

The idea is to... `Keywords: "something", "something_else".`